pkg/kms/webkms/headers_opts.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webkms
+
+import "net/http"
+
+// addHeaders function supports adding http headers to req for a given key server defined in ksConfig.
+type addHeaders func(req *http.Request, ksConfig KeyServerCfg) (*http.Header, error)
```

Comment:
what is  need for ksConfig ?

Reply:
ksConfig should provide the auth server connection config info (url, secret, controller) necessary to obtain a request authorization (eg: zcap signature).

Reply:
thinking a bit more on this, the framework should not be aware of this info, I'll remove this argument from the options, it is needed for integration by the client who needs to authenticate with the key server, so it should be done outside the framework.

Reply:
second argument is now removed